src/ItemAdded.js

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import {Link} from 'react-router-dom';
+import ItemCard from './ItemCard';
+import "./App.css";
+import Nav from './Nav';
+
+const ItemAdded= () => {  
+    return (
+      
+    <section className='main'>
+    <div className='mainContainer'>
+      <Nav />
+        <h1>Item added!</h1>
+        <h3>Other people can now see and borrow your item!</h3>
```

Comment:
It would be nice to redirect a user to his new item or updated list of all items